bootstrap_cfn/fab_tasks.py

```diff
@@ -342,6 +343,10 @@ def cfn_create():
     try:
         stack = cfn.create(stack_name, cfn_config.process())
     except:
+        # cleanup ssl certificates if any
+        if 'ssl' in cfn_config.data:
+            print red("Deleting SSL certificates from stack")
+            iam.delete_ssl_certificate(cfn_config.ssl(), stack_name)
```

Comment:
Can this itself fail? Maybe we need to catch errors here too? If we do we should still fail with the original error.